src/boot/bootctl.c

```diff
@@ -709,21 +738,13 @@ static int status_entries(
                        SD_ID128_FORMAT_VAL(dollar_boot_partition_uuid));
         printf("\n\n");
 
-        r = boot_config_load(&config, esp_path, xbootldr_path);
-        if (r < 0)
-                return r;
-
-        r = boot_config_select_special_entries(&config);
-        if (r < 0)
-                return r;
```

Comment:
hmm, why not call boot_config_load_and_select() from here?

(I mean, i am not opposed to moving this, I am just wondering, given that `status_entries()` only has a single caller)

Reply:
I wanted to keep the loading and printing separate. Looking at it again, either option should work about as well…